src/components/Feed/Feed.tsx

```diff
@@ -38,34 +39,26 @@ const Feed = ({ userSentiment, articles, history, updateUserSentiment, storeArti
         image={ article.multimedia.url }
         id={ article.id }
         sentiment={ article.sentiment }
+        topic={ article.topic }
         updateUserSentiment={ updateUserSentiment }
         storeArticle={ storeArticle }
         key={ article.title }
       />
     )
   })
-
-  return (
-    <>
-      <div className="articles-container">
-        <Link to='/history'>
-          <button className='history-btn'>History</button>
-        </Link>
-        <Link to='/'>
-          <button className='retake-btn'>Retake Questionnaire</button>
```

Comment:
Maybe even using an icon like this for the retake button?
![Screen Shot 2021-11-08 at 8 40 09 AM](https://user-images.githubusercontent.com/72999840/140782629-e4f57c75-8e86-4964-abf1-8f6eb30ee4af.png)